src/tree/registries/UnifiedRegistryTreeDataProvider.ts

```diff
@@ -74,17 +70,17 @@ export class UnifiedRegistryTreeDataProvider implements vscode.TreeDataProvider<
                     continue;
                 }
 
-                unifiedRoots.push(...roots.map(r => {
+                unifiedRegistryItems.push(...roots.map(r => {
                     return {
                         provider,
                         wrappedItem: r,
                         parent: undefined
                     };
                 }));
             }
-
-            return unifiedRoots;
         }
+
+        return unifiedRegistryItems.sort((a, b) => (a.wrappedItem as CommonRegistryItem).label.localeCompare((b.wrappedItem as CommonRegistryItem).label));

```

Comment:
I think we need a more advanced locale compare function. Even though all of our current registry providers use `CommonRegistryItem`, it's not a strict requirement. So what I'd do is try getting their labels if they are `CommonRegistryItems`, but if not, sort by their `toString` results.

Reply:
yeah i like that!

Reply:
should we add a `isCommonRegistryItem()` in the package then?

Reply:
or can we do `instanceof`

Reply:
I'm surprised we haven't already got an `isCommonRegistryItem` method but that would be the way to go.